src/vs/workbench/services/textfile/common/textFileEditorModelManager.ts

```diff
@@ -217,6 +233,12 @@ export class TextFileEditorModelManager implements ITextFileEditorModelManager {
 						break;
 				}
 			});
+
+			// Install model content change listener
+			this.mapResourceToModelContentChangeListener[resource.toString()] = model.onDidContentChange(() => {
+				const newEvent = new TextFileModelChangeEvent(model, StateChange.CONTENT_CHANGE);
```

Comment:
Already tracked in https://github.com/Microsoft/vscode/issues/14186, added a note to make event types more consistent.